client/src/components/CTACard.tsx

```diff
@@ -0,0 +1,263 @@
+import * as React from 'react';
+
+import Grid from '@mui/material/Grid';
+import Avatar from '@mui/material/Avatar';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import { IconButton, Typography } from '@mui/material';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import { makeStyles } from '@mui/styles';
+
+//     backgroundColor: '#37718E', //primary dark blue color
+const useStyles = makeStyles((theme) => ({
+  CTAGrid: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#FFFFFF',
+    paddingTop: '25px',
+  },
+
+  CTACardLayout: {
+    border: '1',
+    borderColor: '#37718E',
+  },
+
+  CTACardContent1: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#37718E',
+    color: '#FFFFFF',
+    fontSize: '18px',
+    fontWeight: 'bold',
+    textAlign: 'left',
+  },
+
+  CTACardContent2: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#7C4164',
+    color: '#FFFFFF',
+    fontSize: '18px',
+    fontWeight: 'bold',
+    textAlign: 'left',
+  },
+
+  CTACardContent3: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#C33D54',
+    color: '#FFFFFF',
+    fontSize: '18px',
+    fontWeight: 'bold',
+    textAlign: 'left',
+  },
+
+  HeaderStyle: {
+    paddingY: '2',
+    my: '2',
+    mt: '2',
+  },
+
+  ParagraphStyle: {
+    textColor: '#FFFFFF',
+    fontSize: '10',
+    textAlign: 'left',
+    width: '80%',
+  },
+
+  CTAButton: {
+    display: 'inline-block',
+    justifyContent: 'bottom',
+    marginTop: '100px',
+    variant: '',
+    width: '60%',
+  },
+
+  BottomCTACard: {
+    display: 'flex',
+  },
+}));
+
+function CTACard() {
+  const classes = useStyles({});
+
+  const loremIpsum =
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Facilisis placerat et, at vel tristique.';
+  const headerText = 'Tertiary Headline Title';
+
+  const [btnText, setBtnText] = React.useState<string>('');
+  const [btnText2, setBtnText2] = React.useState<string>('');
+  const [btnText3, setBtnText3] = React.useState<string>('');
+
```

Comment:
#### TODO
Try finding a way to avoid adding each card statically, you can put all this metadata in an array of objects - each object would have the metadata for each card. To do this, you'll have to create a new component  (can be in the same file) `CallToActionWrapper`. 

The wrapper is where you can define the cards array and then use the `array.map()` method to display each card. 

Here is a sample of what this looks like: https://www.digitalocean.com/community/tutorials/how-to-create-wrapper-components-in-react-with-props

Let me know if you want to pair on this

Reply:
Hi Esteban, working together on this would be super helpful for me. I looked over the article you sent and I think I get what we are trying to accomplish but maybe having a quick chat would help. I am available Thursday afternoon if you are available at that time.

Reply:
Actually, this is a more simplified version of the wrapper/component pattern: https://codesandbox.io/s/jolly-lake-qcqlpl?file=/src/components/CallToAction.jsx